Can't build

Hello
```
./linux_install.sh 
~/projects/Gempyre/build ~/projects/Gempyre
-- The CXX compiler identification is GNU 13.2.0
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- The C compiler identification is GNU 13.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
compiler is GNU
Build is debug
Defaults to Browser UI
-- Using the multi-header code from /home/ubuntu/projects/Gempyre/build/_deps/libjson-src/include/
Retrieve zlib
skip libpfd version, use latest
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD
-- Performing Test CMAKE_HAVE_LIBC_PTHREAD - Success
-- Found Threads: TRUE  
-- Found Doxygen: /usr/bin/doxygen (found version "1.9.8") found components: doxygen dot 
SYSTEM_INCLUDES: /home/ubuntu/projects/Gempyre/build/_deps/libjson-src/include;/home/ubuntu/projects/Gempyre/build/gempyrelib/libsockets-prefix/src/libsockets/src;/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/src;/home/ubuntu/projects/Gempyre/build/gempyrelib/zlib-prefix/src/zlib/src/zlib;/home/ubuntu/projects/Gempyre/build/gempyrelib/libpfd-prefix/src/libpfd;/home/ubuntu/projects/Gempyre/build/_deps/lodepng-src
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_VISIBILITY - Success
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY
-- Performing Test COMPILER_HAS_HIDDEN_INLINE_VISIBILITY - Success
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR
-- Performing Test COMPILER_HAS_DEPRECATED_ATTR - Success
GempyreConfig.cmake is stored in lib/cmake/gempyre
Websocket library is uwebsockets
-- Configuring done (280.9s)
-- Generating done (0.1s)
-- Build files have been written to: /home/ubuntu/projects/Gempyre/build
[  1%] Creating directories for 'libsockets'
[  2%] Performing download step (git clone) for 'libsockets'
Cloning into 'libsockets'...
remote: Enumerating objects: 2675, done.        
remote: Counting objects: 100% (787/787), done.        
remote: Compressing objects: 100% (204/204), done.        
remote: Total 2675 (delta 671), reused 611 (delta 582), pack-reused 1888        
Receiving objects: 100% (2675/2675), 891.32 KiB | 722.00 KiB/s, done.
Resolving deltas: 100% (1763/1763), done.
HEAD is now at 5440dba Apply read-wants-write patch
[  3%] No update step for 'libsockets'
[  4%] No patch step for 'libsockets'
[  5%] No configure step for 'libsockets'
[  6%] No build step for 'libsockets'
[  7%] No install step for 'libsockets'
[  8%] No test step for 'libsockets'
[  9%] Completed 'libsockets'
[  9%] Built target libsockets
[ 10%] Creating directories for 'zlib'
[ 11%] Performing download step (git clone) for 'zlib'
Cloning into 'zlib'...
remote: Enumerating objects: 6634, done.        
remote: Counting objects: 100% (125/125), done.        
remote: Compressing objects: 100% (80/80), done.        
remote: Total 6634 (delta 53), reused 87 (delta 45), pack-reused 6509        
Receiving objects: 100% (6634/6634), 4.25 MiB | 778.00 KiB/s, done.
Resolving deltas: 100% (4715/4715), done.
HEAD is now at cacf7f1 zlib 1.2.11
[ 12%] No update step for 'zlib'
[ 13%] No patch step for 'zlib'
[ 14%] No configure step for 'zlib'
[ 15%] No build step for 'zlib'
[ 16%] No install step for 'zlib'
[ 17%] No test step for 'zlib'
[ 18%] Completed 'zlib'
[ 18%] Built target zlib
[ 19%] Creating directories for 'libwebsockets'
[ 20%] Performing download step (git clone) for 'libwebsockets'
Cloning into 'libwebsockets'...
remote: Enumerating objects: 9869, done.        
remote: Counting objects: 100% (459/459), done.        
remote: Compressing objects: 100% (206/206), done.        
remote: Total 9869 (delta 285), reused 346 (delta 251), pack-reused 9410        
Receiving objects: 100% (9869/9869), 4.62 MiB | 777.00 KiB/s, done.
Resolving deltas: 100% (6583/6583), done.
HEAD is now at 44af0e5 Fix HttpParser/MessageParser for bytes above 7-bit ASCII
Submodule 'fuzzing/libEpollFuzzer' (https://github.com/uNetworking/libEpollFuzzer) registered for path 'fuzzing/libEpollFuzzer'
Submodule 'fuzzing/seed-corpus' (https://github.com/uNetworking/seed-corpus.git) registered for path 'fuzzing/seed-corpus'
Submodule 'libdeflate' (https://github.com/ebiggers/libdeflate) registered for path 'libdeflate'
Submodule 'uSockets' (https://github.com/uNetworking/uSockets.git) registered for path 'uSockets'
Cloning into '/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/fuzzing/libEpollFuzzer'...
Cloning into '/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/fuzzing/seed-corpus'...
Cloning into '/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/libdeflate'...
Cloning into '/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/uSockets'...
Submodule path 'fuzzing/libEpollFuzzer': checked out 'b9bbd2d490a2fe3784283e0e456381ff26f175d6'
Submodule path 'fuzzing/seed-corpus': checked out '69f5fd36c102f0a7d527049983c0e2b2a822c23d'
Submodule path 'libdeflate': checked out '4d3c0f00d55c3bfd7abe3f7ac3d99e3bd0f7f96b'
Submodule path 'uSockets': checked out 'c2c1bbfa1644f1f6eb7fc9375650f41c5f9b7b06'
Submodule 'boringssl' (https://github.com/google/boringssl) registered for path 'uSockets/boringssl'
Cloning into '/home/ubuntu/projects/Gempyre/build/gempyrelib/libwebsockets-prefix/src/libwebsockets/uSockets/boringssl'...
Submodule path 'uSockets/boringssl': checked out '28c48e38a4489e77e2b00fb8ca5d46ea0f21618f'
[ 21%] No update step for 'libwebsockets'
[ 22%] No patch step for 'libwebsockets'
[ 23%] No configure step for 'libwebsockets'
[ 24%] No build step for 'libwebsockets'
[ 25%] No install step for 'libwebsockets'
[ 26%] No test step for 'libwebsockets'
[ 27%] Completed 'libwebsockets'
[ 27%] Built target libwebsockets
[ 28%] Creating directories for 'libpfd'
[ 29%] Performing download step (git clone) for 'libpfd'
Cloning into 'libpfd'...
remote: Enumerating objects: 655, done.        
remote: Counting objects: 100% (274/274), done.        
remote: Compressing objects: 100% (106/106), done.        
remote: Total 655 (delta 171), reused 264 (delta 168), pack-reused 381        
Receiving objects: 100% (655/655), 172.86 KiB | 541.00 KiB/s, done.
Resolving deltas: 100% (407/407), done.
Already on 'main'
Your branch is up to date with 'origin/main'.
[ 30%] No update step for 'libpfd'
[ 31%] No patch step for 'libpfd'
[ 32%] No configure step for 'libpfd'
[ 33%] No build step for 'libpfd'
[ 34%] No install step for 'libpfd'
[ 35%] No test step for 'libpfd'
[ 36%] Completed 'libpfd'
[ 36%] Built target libpfd
[ 37%] Generating src/generated/gempyre.js.h
[ 37%] Built target gempyre_resource
[ 38%] Generating libsockets-prefix/src/libsockets-build/uSockets.a
ar: creating uSockets.a
a - bsd.o
a - context.o
a - epoll_kqueue.o
a - gcd.o
a - libuv.o
a - loop.o
a - openssl.o
a - socket.o
a - wolfssl.o
[ 38%] Built target makecmd
[ 39%] Generate API documentation for doxy_docs
Doxygen version used: 1.9.8
Searching for include files...
Searching for example files...
Searching for images...
Searching for dot files...
Searching for msc files...
Searching for dia files...
Searching for files to exclude
Searching INPUT for files to process...
Reading and parsing tag files
Parsing files
Reading /home/ubuntu/projects/Gempyre/README.md...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre.h...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_bitmap.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_bitmap.h...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_client.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_client.h...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_types.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_types.h...
Preprocessing /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_utils.h...
Parsing file /home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_utils.h...
Building macro definition list...
Building group list...
Building directory list...
Building namespace list...
Building file list...
Building class list...
Building concept list...
Computing nesting relations for classes...
Associating documentation with classes...
Associating documentation with concepts...
Associating documentation with modules...
Building example list...
Searching for enumerations...
Searching for documented typedefs...
Searching for members imported via using declarations...
Searching for included using directives...
Searching for documented variables...
Building interface member list...
Building member list...
Searching for friends...
Searching for documented defines...
Computing class inheritance relations...
Computing class usage relations...
Flushing cached template relations that have become invalid...
Computing class relations...
Add enum values to enums...
Searching for member function documentation...
Creating members for template instances...
Building page list...
Search for main page...
Computing page relations...
Determining the scope of groups...
Computing module relations...
Sorting lists...
Determining which enums are documented
Computing member relations...
Building full member lists recursively...
Adding members to member groups.
Computing member references...
Inheriting documentation...
Generating disk names...
Adding source references...
Adding xrefitems...
Sorting member lists...
Setting anonymous enum type...
Computing dependencies between directories...
Generating citations page...
Counting members...
Counting data structures...
Resolving user defined references...
Finding anchors and sections in the documentation...
Transferring function references...
Combining using relations...
Adding members to index pages...
Correcting members for VHDL...
Computing tooltip texts...
Generating style sheet...
Generating search indices...
Generating example documentation...
Generating file sources...
Generating code for file include/gempyre.h...
Generating code for file include/gempyre_bitmap.h...
Generating code for file include/gempyre_client.h...
Generating code for file include/gempyre_graphics.h...
Generating code for file include/gempyre_types.h...
Generating code for file include/gempyre_utils.h...
Generating file documentation...
Generating docs for file include/gempyre.h...
Generating docs for file include/gempyre_bitmap.h...
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_bitmap.h:25: warning: Member gempyre_graphics_assert(b, x) (macro definition) of file gempyre_bitmap.h is not documented.
Generating docs for file include/gempyre_graphics.h...
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:33: warning: Member CanvasDataPtr (typedef) of namespace Gempyre is not documented.
Generating docs for file include/gempyre_utils.h...
Generating page documentation...
Generating group documentation...
Generating class documentation...
Generating concept documentation...
Generating module documentation...
Generating namespace documentation...
Generating docs for compound Gempyre::Bitmap...
Generating docs for compound Gempyre::CanvasElement...
Generating docs for compound Gempyre::Dialog...
Generating docs for compound Gempyre::Element...
Generating docs for compound Gempyre::Event...
Generating docs for compound Gempyre::EventMap...
Generating docs for compound Gempyre::EventQueue...
Generating docs for compound Gempyre::FrameComposer...
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:156: warning: Member stroke_rect(double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:159: warning: Member clear_rect(double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:162: warning: Member fill_rect(double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:192: warning: Member rect(double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:221: warning: Member draw_image(const std::string &id, double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
/home/ubuntu/projects/Gempyre/gempyrelib/include/gempyre_graphics.h:224: warning: Member draw_image(const std::string &id, double cx, double cy, double cw, double ch, double x, double y, double w, double h) (function) of class Gempyre::FrameComposer is not documented.
Generating docs for compound Gempyre::IdList...
Generating docs for compound Gempyre::Rect...
Generating docs for compound Gempyre::Ui...
Generating docs for namespace Gempyre::Color
Generating docs for compound GempyreUtils::LogWriter...
Generating graph info page...
Generating directory documentation...
Generating index page...
/home/ubuntu/projects/Gempyre/README.md:83: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:90: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:96: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:99: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:103: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:136: warning: Illegal </br> tag found
/home/ubuntu/projects/Gempyre/README.md:139: warning: Illegal </br> tag found
Generating main page...
Generating page index...
Generating topic index...
Generating module index...
Generating module member index...
Generating namespace index...
Generating namespace member index...
Generating concept index...
Generating annotated compound index...
Generating alphabetical compound index...
Generating hierarchical class index...
Generating graphical class hierarchy...
Generating member index...
Generating file index...
Generating file member index...
Generating example index...
finalizing index lists...
writing tag file...
Running plantuml with JAVA...
Running dot...
Generating dot graphs using 5 parallel threads...
Running dot for graph 1/22
Running dot for graph 2/22
Running dot for graph 3/22
Running dot for graph 4/22
Running dot for graph 5/22
Running dot for graph 6/22
Running dot for graph 7/22
Running dot for graph 8/22
Running dot for graph 9/22
Running dot for graph 10/22
Running dot for graph 11/22
Running dot for graph 12/22
Running dot for graph 13/22
Running dot for graph 14/22
Running dot for graph 15/22
Running dot for graph 16/22
Running dot for graph 17/22
Running dot for graph 18/22
Running dot for graph 19/22
Running dot for graph 20/22
Running dot for graph 21/22
Running dot for graph 22/22
Patching output file 1/30
Patching output file 2/30
Patching output file 3/30
Patching output file 4/30
Patching output file 5/30
Patching output file 6/30
Patching output file 7/30
Patching output file 8/30
Patching output file 9/30
Patching output file 10/30
Patching output file 11/30
Patching output file 12/30
Patching output file 13/30
Patching output file 14/30
Patching output file 15/30
Patching output file 16/30
Patching output file 17/30
Patching output file 18/30
Patching output file 19/30
Patching output file 20/30
Patching output file 21/30
Patching output file 22/30
Patching output file 23/30
Patching output file 24/30
Patching output file 25/30
Patching output file 26/30
Patching output file 27/30
Patching output file 28/30
Patching output file 29/30
Patching output file 30/30
type lookup cache used 1343/65536 hits=6298 misses=1516
symbol lookup cache used 1128/65536 hits=6654 misses=1128
finished...
[ 39%] Built target doxy_docs
[ 40%] dictionary
[ 41%] Spellchecking
Spelling errors:
Looking for enums:
/home/ubuntu/projects/Gempyre/scripts/gempyre_func_name.py:57:                               '__enums': []}
Fix 'enums' or add into dictionary!
Looking for typedefs:
not found typedefs
Fix 'typedefs' or add into dictionary!
gmake[2]: *** [gempyrelib/CMakeFiles/doxy_spell_bin.dir/build.make:75: gempyrelib/CMakeFiles/doxy_spell_bin] Error 1
gmake[1]: *** [CMakeFiles/Makefile2:1361: gempyrelib/CMakeFiles/doxy_spell_bin.dir/all] Error 2
gmake: *** [Makefile:146: all] Error 2

```

--- a/aspell/gempyre_words.txt
+++ b/aspell/gempyre_words.txt
@@ -42,6 +42,7 @@ dllexport
 doxygen
 endif
 enum
+enums
 env
 eval
 filemap
@@ -290,6 +291,7 @@ textBaseline
 tolower
 txt
 typedef
+typedefs
 Typedef
 Typedefs
 typename

